cmd/boulder-va/gsb.go

```diff
@@ -64,7 +75,7 @@ func (sb gsbAdapter) IsListed(url string) (string, error) {
 	if err != nil {
 		return "error", err
 	}
-	if len(threats) > 0 {
+	if len(threats) > 0 && threats[0] != nil {
```

Comment:
Got it, that makes sense, though the code is confusing. What about:

```
if len(threats) > 0 && len(threats[0]) > 0 {
  threats[0][0].ThreatType.String()
} else if len(threats) > 0 { 
 EmptyURLThreatErr
}
```

Note that we don't really use the threat type AFAIK so we could also just make up a threat type and return it anytime `len(threats) > 0`.